examples/connector/java/src/main/java/connector/ConnectorServiceImpl.java

```diff
@@ -17,22 +17,22 @@ public void configurationForm(ConfigurationFormRequest request, StreamObserver<C
                         .setTableSelectionSupported(true)
                         .addAllFields(Arrays.asList(
                                 FormField.newBuilder()
-                                        .setName("apikey").setLabel("API key").setRequired(true).setTextField(TextField.PlainText).build(),
+                                        .setSingle(Field.newBuilder().setName("apiKey").setLabel("API Key").setRequired(true).setTextField(TextField.PlainText).build())
+                                        .build(),
```

Comment:
Let's also use the new `placeholder` and `default_value` fields in the example

Reply:
added